hypha/apply/projects/models/report.py

```diff
@@ -182,19 +182,44 @@ class ReportConfig(models.Model):
 
     WEEK = _('week')
     MONTH = _('month')
+    YEAR = _('year')
+    ONE_TIME = _('one_time')
     FREQUENCY_CHOICES = [
+        ('', '---'),
+        (ONE_TIME, _('One Time')),
         (WEEK, _('Weeks')),
         (MONTH, _('Months')),
+        (YEAR, _('Years')),
     ]
```

Comment:
Consider switching to `TextChoices` see this implementation as an example. 

(https://github.com/HyphaApp/hypha/blob/main/hypha/apply/activity/options.py#L5).